incubator/virtualcluster/pkg/syncer/resources/register.go

```diff
@@ -58,6 +60,16 @@ func init() {
 	}
 }
 
+// extraResourceToController maps extra resource name with extra resource controller
```

Comment:
how about creating a map for these controller?  controller-name -> func
check if enable this controller by `ExtraSyncingResources`
just like kcm https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-controller-manager/app/controllermanager.go#L351

Reply:
sure